jakeisnt/verde
# use nodemon to hot reload server

closes #49 
to test:
- reinstall dependencies
- start server
- edit file in server/ dir and make sure it reloads (adding console.log easy way to do this)
